usmqe/web/tendrl/cluster_work.py

```diff
@@ -0,0 +1,230 @@
+"""
+Description: Simple cluster auxiliary methods and classes
+
+Author: ltrilety
+"""
+
+import pytest
+
+from webstr.selenium.ui.support import WebDriverUtils
+from usmqe.web.tendrl.mainpage.navpage.pages import NavMenuBars
+from usmqe.web.tendrl.mainpage.clusters.cluster_list.pages import\
+    ClustersList, ClustersMenu
+from usmqe.web.tendrl.landing_page.pages import get_landing_page
+from usmqe.web.tendrl.auxiliary.pages import UpperMenu
+from usmqe.web.tendrl.mainpage.clusters.import_cluster_wizard.pages\
+    import ImportCluster
+from usmqe.web.tendrl.mainpage.tasks.pages import TaskDetails
+from usmqe.web.tendrl.mainpage.clusters.cluster.pages import ClusterMenu
+from usmqe.web.tendrl.task_wait import task_wait
+from usmqe.web.tendrl.clusters import check_hosts
+
+
+IMPORT_TIMEOUT = 3600
+
+
+def import_cluster_wait(driver, import_task_details):
+    """
+    wait till the import cluster task is finished
+
+    Parameters:
+        driver: selenium driver
+        import_task_details: webstr page object with import_cluster method
+                       landing_page-HomePage or cluster_list-ClustersMenu
+        ttl (int): how long it waits till the tasks is finished
```

Comment:
That's not even python type :-)
If you ask python for the type, it says `webstr.selenium.webdriver.Firefox` in other words `selenium driver` and for the other it says e.g. `usmqe.web.tendrl.mainpage.navpage.pages.NavMenuBars` in other words `instance of some class`
Anyway if we agreed we should use `driver` and `object` for those I will change that. I don't mind.